src/Processor/FillMissingProcessor.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the BibTex Parser.
+ *
+ * (c) Renan de Lima Barbosa <renandelima@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace RenanBr\BibTexParser\Processor;
+
+class FillMissingProcessor
+{
+    use TagCoverageTrait;
```

Comment:
We don't need this trait, as this processor won't change any pre-existing tag.

Reply:
I wasn't sure about that one, should have asked :smile: